frontend/src/pages/ProductDetail/Selection/Selection.scss

```diff
@@ -0,0 +1,56 @@
+@import '@assets/scss/mixins.scss';
+@import '@assets/scss/variables.scss';
+
+.selection {
+  padding: 0 0 16px;
+  &__list {
+    @include flex(center, flex-start);
+    padding: 16px 0px;
+  }
+  &__item {
+    @include flex(center, center);
+    height: 48px;
+    width: 48px;
+    margin: 0 11px 0 0;
+    padding: 0 1px;
+    border: $gray-primary 1px solid;
+    border-radius: 50%;
+    transition: all 0.1s ease-in-out;
+    &:hover {
+      @include zoom_animation;
+    }
+    &--active {
+      border: white 2px solid !important;
+    }
+  }
+  &-color {
```

Comment:
I remember you define this before, if you make sure it is redundant then you should move this to common CSS